invenio_app_ils/ui/backoffice/src/common/api/date.js

```diff
@@ -0,0 +1,13 @@
+import { DateTime } from 'luxon';
+
+export const fromBackend = stringDate => {
+  return DateTime.fromISO(stringDate);
+};
+
+export const toBackend = date => {
```

Comment:
toISO? I know is about readability but...

Reply:
Done!